db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryCache.java

```diff
@@ -130,6 +132,7 @@ public void onRemoval(RemovalNotification<Object, Dependent> removalNotification
         roleCache = new ManagedCache<>(CacheBuilder.newBuilder().recordStats().maximumSize(100).build());
```

Comment:
Do we need to make these cache sizes configurable? If not in this PR, maybe it's better to revisit all our settings on cache.

Reply:
@wtang16 Yes, currently, only 5 cache sizes are configurable, the remaining(8 caches) are not. I've opened SPLICE-2119 to make all of them configurable.